pkg/component/runtime/command.go

```diff
@@ -243,11 +260,26 @@ func (c *CommandRuntime) start(comm Communicator) error {
 		return nil
 	}
 	cmdSpec := c.current.Spec.Spec.Command
-	env := make([]string, 0, len(cmdSpec.Env))
+	env := make([]string, 0, len(cmdSpec.Env)+2)
 	for _, e := range cmdSpec.Env {
 		env = append(env, fmt.Sprintf("%s=%s", e.Name, e.Value))
 	}
-	proc, err := process.Start(c.current.Spec.BinaryPath, os.Geteuid(), os.Getgid(), cmdSpec.Args, env, attachOutErr)
+	env = append(env, fmt.Sprintf("%s=%s", envAgentComponentID, c.current.ID))
+	env = append(env, fmt.Sprintf("%s=%s", envAgentComponentInputType, c.current.Spec.InputType))
+	uid, gid := os.Geteuid(), os.Getgid()
+	workDir, err := c.workDir(uid, gid)
+	if err != nil {
+		return err
```

Comment:
more context please.

Reply:
`workDir` already adds the context information to the error.